Subject: [PATCH] REVERT ME
---
Index: examples/basic-externals/src/storyshots/ui.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/examples/basic-externals/src/storyshots/ui.ts b/examples/basic-externals/src/storyshots/ui.ts
--- a/examples/basic-externals/src/storyshots/ui.ts	(revision 6fcea86e5827f989aaddb6a7fbf9c0fcc60f175c)
+++ b/examples/basic-externals/src/storyshots/ui.ts	(revision df59337fe35db9d74f72b27b7e96053137e707ef)
@@ -1,4 +1,4 @@
 import { config } from './config';
-import { runUI } from '@storyshots/manager';
+import { runUI } from '../../../../packages/manager/src';
 
 void runUI(config);
Index: packages/manager/src/server/modes/createManager.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/packages/manager/src/server/modes/createManager.ts b/packages/manager/src/server/modes/createManager.ts
--- a/packages/manager/src/server/modes/createManager.ts	(revision 6fcea86e5827f989aaddb6a7fbf9c0fcc60f175c)
+++ b/packages/manager/src/server/modes/createManager.ts	(revision df59337fe35db9d74f72b27b7e96053137e707ef)
@@ -1,9 +1,20 @@
 import express from 'express';
 import path from 'path';
 import { root } from '../compiler/manager-root';
+import managerConfig from '../compiler/manager-config';
+import { webpack } from 'webpack';
+import dev from 'webpack-dev-middleware';
+
+function createHOTManager() {
+  console.log('createHOTManager');
+
+  return dev(webpack(managerConfig));
+}
 
 function createPrebuiltManager() {
+  console.log('createPrebuiltManager');
+
   return express.static(path.join(root, 'lib', 'client'));
 }
 
-export const createManager = createPrebuiltManager;
+export const createManager = createHOTManager;
